### ISSUE TITLE
No devices delivered from cloud

### ISSUE THREAD

**Issue by christof62** [ReneNulschDE/ha-mysmartbike#1]
No sensor entities are created.
Smartphone App is connected with bike.
Bike: ORBEA Gain 170 EU

Home assistant core: 2024.4.4

Debug log:
```
2024-04-27 19:58:35.903 DEBUG (MainThread) [custom_components.mysmartbike] Start async_unload_entry
2024-04-27 19:58:35.905 DEBUG (MainThread) [custom_components.mysmartbike.webapi] login: Start
2024-04-27 19:58:35.905 DEBUG (MainThread) [custom_components.mysmartbike.webapi] login: Token not expired
2024-04-27 19:58:35.905 DEBUG (MainThread) [custom_components.mysmartbike.coordinator] Start save_token
2024-04-27 19:58:35.906 DEBUG (MainThread) [custom_components.mysmartbike.coordinator] _async_update_data: started
2024-04-27 19:58:36.218 INFO (MainThread) [custom_components.mysmartbike.coordinator] MySmartBike Cloud delivered 0 device(s).
2024-04-27 19:58:36.218 DEBUG (MainThread) [custom_components.mysmartbike.coordinator] {}
2024-04-27 19:58:36.218 DEBUG (MainThread) [custom_components.mysmartbike.coordinator] Finished fetching mysmartbike data in 0.312 seconds (success: True)
2024-04-27 19:58:36.219 DEBUG (MainThread) [custom_components.mysmartbike.sensor] async_setup_entry: Sensor count for creation - 0
2024-04-27 19:58:36.220 DEBUG (MainThread) [custom_components.mysmartbike.device_tracker] async_setup_entry: DeviceTracker count for creation - 0
```

**Comment by ReneNulschDE** [comment]
Thanks for reporting. Could you please go to [my-smartbike.com.](https://my-smartbike.com/), login with the same credentials like in HA. Click on "My ebikes" on the left side and check if bikes are listed there?

**Comment by christof62** [comment]
Thanks for the fast response.
I checked this already before. Yes, my bike is listed there and all data is shown.
There is also a "devices" tab, where the Mahle components are listed:
- MAHLE X20 Joystick Orbea
- MAHLE X20 Controller Orbea Gain EU 170
- MAHLE X20 iX350 Orbea
- MAHLE X20 Motor

**Comment by ReneNulschDE** [comment]
Thx, could you go to the same page again. Switch on the developer tools of the browser with F12. Switch to the network tab in this tools window and then refresh the page. 

You will see a lot of requests. At the end minus 5 or 6 you should see two requests to the "me" end point. The second one should have all the bike details. Can you send the results to MySmartBike.i1@nulsch.de, please.

This gives me the chance to check why your bike is not recognized. 

Thx
René

**Comment by ReneNulschDE** [comment]
Discussed via email. Looks like that bikes connected via android devices are not visible. Needs further tests. I'll publish a test hotfix later this week. Thanks to @christof62 and his son for the intensive testing.

**Comment by ReneNulschDE** [comment]
Should be fixed with version v0.2.0 - please test it again.

**Comment by christof62** [comment]
It's working now.   😄 
Thanks!
